src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/Server/SmiMetaData.cs

```diff
@@ -942,6 +964,9 @@ ParameterDirection direction
             byte scale,
             long localeId,
             SqlCompareOptions compareOptions,
+#if NET6_0_OR_GREATER
```

Comment:
I don't like that this isn't clear that it applies to the parameter following the #endif but i can't see another way to format the code to make it clear :(

Reply:
Yeah, that's unfortunate.